tests/testsuite/lints/implicit_features/edition_2021/mod.rs

```diff
@@ -23,12 +23,10 @@ bar = { version = "0.1.0", optional = true }
         .build();
 
     snapbox::cmd::Command::cargo_ui()
-        .masquerade_as_nightly_cargo(&["always_nightly"])
         .current_dir(p.root())
         .arg("check")
-        .arg("--quiet")
```

Comment:
Why drop `--quiet`?

Reply:
It would've been an empty file, which does show that the lint wouldn't have run, but it felt odd. This should probably be switched to a non-UI test in the long run.